### DISCUSSION TITLE
Cant start as port 22 in use

### DISCUSSION THREAD

**Discussion by matt-seedata-io** [mariocandela/beelzebub#37] [Q&A] [answered]
Im trying to runBeelzebub in a docker container under multipass (meaning that i have to use port 22 to ssh onto the docker server)... I've removed the port 22 ssh listener from configurations/services. I still get a conflict regarding port 22 when I run docker-compose up

ubuntu@docker:~/beelzebub$ docker-compose up
WARNING: The RABBITMQ_URI variable is not set. Defaulting to a blank string.
Starting beelzebub ... 
Starting beelzebub ... error

ERROR: for beelzebub  Cannot start service beelzebub: driver failed programming external connectivity on endpoint beelzebub (fe96c7de08cd0b78a9e889273d757e6698a3e4510ee432fb4646df6100cd8c34): Error starting userland proxy: listen tcp4 0.0.0.0:22: bind: address already in use

ERROR: for beelzebub  Cannot start service beelzebub: driver failed programming external connectivity on endpoint beelzebub (fe96c7de08cd0b78a9e889273d757e6698a3e4510ee432fb4646df6100cd8c34): Error starting userland proxy: listen tcp4 0.0.0.0:22: bind: address already in use
ERROR: Encountered errors while bringing up the project.

**Comment by mariocandela** [comment, marked as answer]
Hello Matt, 

Nice to meet you :) 

You could delete config file: ssh-22.yaml and remove the port bind 22 from the [docker-compose.yml](https://github.com/mariocandela/beelzebub/blob/main/docker-compose.yml#L10).

I look forward to your kind reply

Thanks for your time 

Cheers 

Mario Candela
